src/applications/mhv/medical-records/containers/LandingPage.jsx

```diff
@@ -292,10 +292,7 @@ const LandingPage = () => {
                   </p>
                   <p className="vads-u-margin-bottom--2">
                     <a
-                      href={mhvUrl(
-                        isAuthenticatedWithSSOe(fullState),
-                        'secure-messaging',
-                      )}
+                      href="http://www.mhv-syst.myhealth.va.gov/mhv-portal-web/compose-message"
```

Comment:
With MHV deep-linking, this will actually be:
`href={mhvUrl(
                        isAuthenticatedWithSSOe(fullState),
                        'compose-message',
                      )}`

Reply:
I've addressed this.